ign_ros2_control/src/ign_system.cpp

```diff
@@ -590,14 +599,14 @@ hardware_interface::return_type IgnitionSystem::write(
       }
     }
 
-    if (this->dataPtr->joints_[i].joint_control_method & POSITION) {
+    else if (this->dataPtr->joints_[i].joint_control_method & POSITION) {
```

Comment:
else-if so the velocity command from this interface doesn't conflict with the one at L597